student-work/assignment-6/Anthony-SevillaMeza/ClimbingStairs.py

```
@@ -0,0 +1,40 @@
+class Solution:
+    #fibonacci
+    def climbStairs(self, n: int) -> int:
+        distinct = []
+        
+        for i in range(n):
```

Ruff (0.14.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)